product/reports/180_Configuration Management - Instances/001 All Public Clouds VMs.yaml

```diff
@@ -0,0 +1,57 @@
+---
+- MiqReport:
```

Comment:
@vestival the nesting is wrong. Remove the `MiqReport:` line and indent the other lines to the start of the line and then it passed